views/layout/basic-card.html

```diff
@@ -0,0 +1,7 @@
+{% from "components/basic-card.html" import basicCard %}
```

Comment:
Until [embed](https://github.com/mozilla/nunjucks/issues/790) lands, the best way to create a sub-layout is to create an include, like you are, but feed data in to it via `{% set %}` before it gets called, and have that be used generate the content, looping over the set content.

Here, you're hard-coding this layout to only use the Style Guide's sample data, which we don't want to do.

(Twig's [embed](http://twig.sensiolabs.org/doc/tags/embed.html) function, for reference)